src/core/ext/filters/http/message_compress/compression_filter.cc

```diff
@@ -273,8 +273,8 @@ ArenaPromise<ServerMetadataHandle> ClientCompressionFilter::MakeCallPromise(
         return std::move(*r);
       });
   // Run the next filter, and race it with getting an error from decompression.
-  return Race(decompress_err->Wait(),
-              next_promise_factory(std::move(call_args)));
+  return PrioritizedRace(decompress_err->Wait(),
```

Comment:
How about adding a TODO here that this no longer needs to be prioritized once we switch to the new 5-pipe scheme?